torchmetrics/__init__.py

```diff
@@ -54,9 +54,11 @@
 from torchmetrics.image import (  # noqa: E402
     PSNR,
     SSIM,
+    UQI,
```

Comment:
```suggestion
```

As @ashutoshml pointed out, `UQI` is a new metric for `v0.8` hence there is no need to introduce `UQI` next to `UniversalImageQualityIndex`